site2/website/versions.json

```diff
@@ -1,4 +1,6 @@
 [
+  "2.9.1",
```

Comment:
We'd better not include 2.9.1 now since it is not officially released, right?
Similar to https://github.com/apache/pulsar/pull/10666#discussion_r638651623

Reply:
the VOTE is closed.
I have released all of the artifacts.
we are ready to update the website